packages/lit-dev-content/site/docs/v3/tools/development.md

```diff
@@ -78,6 +78,45 @@ class MyElement extends LitElement {
 
 It's best for code size if the code to control warnings is eliminated in your own production builds.
 
+#### Multiple versions of Lit warning {#multiple-lit-versions}
+
+If Lit is being used as an internal dependency of elements, elements can use different versions of Lit and are completely interoperable.
```

Comment:
We should be clear what triggers the warning at the top of this section, before going into why it's okay with Lit's behavior.

```md
A dev mode only warning is triggered when multiple versions, or even multiple copies of the same version, of any of the Lit core packages – `lit-html`, `lit-element`, `@lit/reactive-element` – are detected.
```

Reply:
Wonderful! Added verbatim.